lib/builder/webpack/client.js

```diff
@@ -164,6 +144,28 @@ export default class WebpackClientConfig extends WebpackBaseConfig {
       }
     }
 
+    // Make uglifyjs faster
+    if (!this.options.dev && !config.optimization.minimizer) {
+      // https://github.com/webpack-contrib/uglifyjs-webpack-plugin
+      const sourceMapVariants = ['#source-map', '@source-map', 'source-map']
+      const sourceMap = sourceMapVariants.includes(config.devtool)
```

Comment:
The official way should be more accurate.
https://github.com/webpack/webpack/blob/master/lib/WebpackOptionsDefaulter.js#L294

Reply:
@clarkdo hm, i though about implementing same thing as you linked, but it will match for example also #eval-source-map and all other source-map variants. And because when i tested with #eval-source-map - all seems to be working even without this pr ( as described in linked issue), so i didnt go that way. 
but since its in official repo i will change it

Reply:
Thank you.